### ISSUE TITLE
do extended cliconn snotes for far connects too

### ISSUE THREAD

**Issue by jesopo** [solanum-ircd/solanum#138]
this could either just be an addition to `sno_farconnect` or we could make the snotes netwide

**Comment by examknow** [comment]
Am I the only one who thinks that all cliconns should be netwide?

**Comment by edk0** [comment]
No, but making them netwide feels kind of bad because connects are already propagated globally

**Comment by jesopo** [comment]
then we need to propagate stuff like connection class.

**Comment by edk0** [comment]
unless it's substantially more than just connection class that we need to propagate I'm inclined to say we should do so. other things that report classes probably deserve to be global too, after all

**Comment by jesopo** [comment]
what about i-line spoof when the user has got a cloak through sasl?

**Comment by 1125f3da-6abf-4975-9a7d-841cc7f1b38e** [comment]
5877 DE11 73EA 08AD 0377 A662 DCBB 099F 3273 976B
